### PR TITLE
o-error integration

### PR DESCRIPTION
**Pull request by expe**

### CODE REVIEW

**Comment by AlbertoElias** [comment]
One of the Travis builds failed, seems to be a network issue, could you restart it?

File: main.js [hunk 1]
```diff
@@ -184,7 +184,12 @@ Tracking.prototype._getDeclarativeConfig = function(options) {
 	try {
 		declarativeOptions = JSON.parse(declarativeConfigString);
 	} catch(e) {
-		throw new Error('Invalid JSON configuration syntax, check validity for o-tracking configuration: "' + e.message + '"');
+		var configError = new Error('Invalid JSON configuration syntax, check validity for o-tracking configuration: "' + e.message + '"');
+		this.utils.broadcast(document.body, 'oErrors', 'log', {
```
Comment (AlbertoElias) [inline main.js]: you haven't removed document.body from the param list

**Comment by expe** [comment]
Cool, just removed the document.body and Travis build is passing.

**Comment by AlbertoElias** [comment]
@markstephens could you double check this is ok?

**Comment by markstephens** [comment]
Yeah, looks ok.

**Pull request merged**